storage/trace_test.go

```diff
@@ -16,251 +16,142 @@ package storage
 
 import (
 	"context"
+	"net/http"
 	"testing"
 
 	"cloud.google.com/go/internal/testutil"
 	"cloud.google.com/go/storage/internal"
 	"github.com/google/go-cmp/cmp"
-	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
+	otcodes "go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/sdk/instrumentation"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	"go.opentelemetry.io/otel/sdk/trace/tracetest"
-	"go.opentelemetry.io/otel/trace"
-	"google.golang.org/api/option"
+	"google.golang.org/api/googleapi"
 )
 
-type emulatedTraceTest struct {
-	*testing.T
-	resources resources
-}
-
-func TestTraceStorageTraceStartEndSpan(t *testing.T) {
+func TestStorageTraceStartEndSpan(t *testing.T) {
 	ctx := context.Background()
-	e := tracetest.NewInMemoryExporter()
-	tp := sdktrace.NewTracerProvider(sdktrace.WithSyncer(e))
-	defer tp.Shutdown(ctx)
-	otel.SetTracerProvider(tp)
+	te := testutil.NewOpenTelemetryTestExporter()
```

Comment:
Can you explain this change/refactor a bit?

Reply:
I've refactored to use the `testutil` function directly. It provides the same functionality out of the box, simplifying our test setup and reducing redundancy.